roxctl/deployment/check/check.go

```diff
@@ -172,6 +175,12 @@ func (d *deploymentCheckCommand) Construct(_ []string, cmd *cobra.Command, f *pr
 }
 
 func (d *deploymentCheckCommand) Validate() error {
+	if d.cluster != "" && d.namespace == defaultNamespace {
+		d.env.Logger().WarnfLn("No custom namespace provided. Any deployments without an explicit namespace will be defaulted to \"default\". Overwrite the default with --namespace/-n [NAMESPACE]\n")
+	}
+	if d.cluster == "" && d.namespace != defaultNamespace {
+		d.env.Logger().WarnfLn("Namespace has been set, but cluster remains unset. Setting the namespace only makes sense when also setting the cluster with '--cluster [CLUSTER]'.\n")
```

Comment:
Let's simplify this a bit and get rid of `f` and `Ln` from `WarnfLn`.
```suggestion
		d.env.Logger().Warn("Cluster is empty. Namespace can be used only if '--cluster' is defined.")
```

Reply:
Done